### PR TITLE
TD-2865 improve error messages when bulk upload excel file doesn't contain correct worksheet

### PR DESCRIPTION
**Pull request by sarathlal-sarangadharan**
### JIRA link
https://hee-tis.atlassian.net/browse/TD-2865

### Description
Added an error validation if  bulk upload excel file doesn't contain correct worksheet **_"DelegatesBulkUpload”_**

### Screenshots
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors (see [info on Text Editor settings](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546185813/DLS+Dev+Process) to avoid whitespace changes)
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript
- [ ] Tested any Views or partials created or changed with [Wave Chrome plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related) and addressed any valid accessibility issues
- [ ] Updated/added documentation in [Confluence](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546939432/DLS+Code) and/or [GitHub Readme](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/README.md). List of documentation links added/changed:
  - [doc_1_here](link_1_here)
- [x] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [ ] Scanned over my pull request in GitHub and addressed any warnings from the GitHub Build and Test checks.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Helpers/CommonValidationErrorMessages.cs [hunk 1]
```diff
@@ -35,5 +35,6 @@ public static class CommonValidationErrorMessages
             "A user with this email address is already registered";
         public const string CentreNameAlreadyExist = "The centre name you have entered already exists, please enter a different centre name";
         public const string MaxBulkUploadRowsLimit = "File must contain no more than {0} rows";
+        public const string InvalidExcelValidationMessage = "The uploaded file must contain a \"DelegatesBulkUpload\" worksheet. Use the \"Download delegates\" button to generate a template.";
```
Comment (kevwhitt-hee) [inline DigitalLearningSolutions.Web/Helpers/CommonValidationErrorMessages.cs:38]: I'd be tempted to make this string property name "InvalidBulkUploadExcelFile" to be more specific and match the pattern of others.

**Review by kevwhitt-hee — APPROVED** [review]
one tiny tweak suggested

**Review by kevwhitt-hee — CHANGES_REQUESTED** [review]
one tiny tweak suggested

File: DigitalLearningSolutions.Web/Helpers/CommonValidationErrorMessages.cs (hunk 1 above)
Comment (sarathlal-sarangadharan) [reply DigitalLearningSolutions.Web/Helpers/CommonValidationErrorMessages.cs]: String property name changed to "InvalidBulkUploadExcelFile" . Please review

**Comment by sarathlal-sarangadharan** [comment]
> one tiny tweak suggested

@kevwhitt-hee String property name changed to "InvalidBulkUploadExcelFile" .Please review.

**Review by kevwhitt-hee — APPROVED** [review]

**Pull request merged**